### PR TITLE
Add filter buttons to search

### PR DESCRIPTION
**Pull request by stormcloud266**
Screenshot 2024-05-08 at 2 02 44 PM

### CODE REVIEW

**Comment by stormcloud266** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/396?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#396** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/396?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#395** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/395?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `dev`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @stormcloud266 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/app/features/activity/screen.tsx [hunk 1]
```diff
@@ -169,9 +180,27 @@ function SearchResults() {
           y: -10,
         }}
       >
+        <XStack gap="$size.0.75">
```
Comment (0xBigBoss) [inline packages/app/features/activity/screen.tsx:183]: if there's only one kind of result, let's not show the filters

**Review by 0xBigBoss — APPROVED** [review]
jsut one comment for me. gj

**Review by 0xBigBoss — APPROVED** [review]

**Comment by stormcloud266** [comment]
### Merge activity

* **May 10, 11:15 PM EDT**: @stormcloud266 started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/396).

**Pull request merged**